tests/conftest.py

```diff
@@ -2899,3 +2908,117 @@ def fixture_demultiplexed_flow_cells_tmp_directory(tmp_path) -> Path:
     tmp_dir = Path(tmp_path, "tmp_run_dir")
 
     return Path(shutil.copytree(original_dir, tmp_dir))
+
+
+## Taxprofiler fixtures
+
+
+@pytest.fixture(scope="session")
+def taxprofiler_config(taxprofiler_dir: Path, taxprofiler_case_id: str) -> None:
+    """Create CSV sample sheet file for testing."""
+    Path.mkdir(Path(taxprofiler_dir, taxprofiler_case_id), parents=True, exist_ok=True)
+    Path(taxprofiler_dir, taxprofiler_case_id, f"{taxprofiler_case_id}_samplesheet").with_suffix(
+        FileExtensions.CSV
+    ).touch(exist_ok=True)
+
+
+@pytest.fixture(scope="session", name="taxprofiler_case_id")
+def fixture_taxprofiler_case_id() -> str:
+    """Returns a taxprofiler case id."""
+    return "taxprofiler_case"
+
+
+@pytest.fixture(scope="session", name="taxprofiler_sample_id")
+def fixture_taxprofiler_sample_id() -> str:
+    """Returns a Taxprofiler sample id."""
+    return "taxprofiler_sample"
+
+
+@pytest.fixture(scope="session", name="taxprofiler_dir")
+def fixture_taxprofiler_dir(tmpdir_factory, apps_dir: Path) -> Path:
+    """Return the path to the Taxprofiler directory."""
+    taxprofiler_dir = tmpdir_factory.mktemp("taxprofiler")
+    return Path(taxprofiler_dir).absolute()
+
+
+@pytest.fixture(scope="session", name="taxprofiler_housekeeper_dir")
+def fixture_taxprofiler_housekeeper_dir(tmpdir_factory, taxprofiler_dir: Path) -> Path:
+    """Return the path to the Taxprofiler Housekeeper bundle directory."""
+    return tmpdir_factory.mktemp("bundles")
+
+
+@pytest.fixture(scope="session", name="taxprofiler_fastq_file_forward")
+def fixture_taxprofiler_fastq_file_l_1_r_1(taxprofiler_housekeeper_dir: Path) -> Path:
+    return Path(taxprofiler_housekeeper_dir, "forward_read.fastq.gz")
+
+
+@pytest.fixture(scope="session", name="taxprofiler_fastq_file_reverse")
+def fixture_taxprofiler_fastq_file_l_1_r_2(taxprofiler_housekeeper_dir: Path) -> Path:
+    return Path(taxprofiler_housekeeper_dir, "reverse_read.fastq.gz")
+
+
+@pytest.fixture(scope="session", name="taxprofiler_mock_fastq_files")
+def fixture_taxprofiler_mock_fastq_files(
+    taxprofiler_fastq_file_forward: Path, taxprofiler_fastq_file_reverse: Path
```

Comment:
can these be general? why do we need fastq fixture that are pipeline specific?

Reply:
I was thinking if I can make them general in this [PR](https://github.com/Clinical-Genomics/cg/issues/2335) instead.

Reply:
sorry I missed your comment about doing this in a different PR. You can disregard this then